drivers/serial/uart_stm32.c

```diff
@@ -28,6 +28,14 @@
 #include <drivers/clock_control/stm32_clock_control.h>
 #include "uart_stm32.h"
 
+#include <stm32_ll_usart.h>
+#include <stm32_ll_lpuart.h>
```

Comment:
Several `soc.h` don't include `stm32*_ll_lpuart.h` header files at the moment, is this handled with dummy/empty include files in the `stm32_hal` repo ?

Reply:
@lowlander Yes. Generated headers only contain entries for families that have the API.

https://github.com/gmarull/hal_stm32/blob/5b5cfa63fa9b9f5f648bc20e1500ab377af85af8/include/stm32_ll_lpuart.h